flower-shop/flower_shop.py

```diff
@@ -0,0 +1,67 @@
+flowers_list = []
+bouquets = []
+order_list = []
+
+
+class Flower:
+    def __init__(self, name, color):
+        self.name = name
+        self.color = color
+
+    def save_flower(self):
+        flower = {'type': self.name, 'color': self.color}
+        flowers_list.append(flower)
+
+
+class Bouquet:
+    def __init__(self, name, price, quantity, *flower_types):
+        self.name = name
+        self.price = price
+        self.quantity = quantity
+        self.flower_types = flower_types
+
+    def create_bouquet(self):
+        bouquet = []
+        for flower in flowers_list:
+            for flower_type in self.flower_types:
+                if flower['type'] == flower_type:
+                    bouquet.append(flower)
+        bouquet_dict = {'bouquet_name': self.name, 'quantity': self.quantity, 'price': self.price, 'bouquet': bouquet}
+        bouquets.append(bouquet_dict)
+
+
+class Shop:
+    def __init__(self, bouquet_name, ordered_quantity):
+        self.bouquet_name = bouquet_name
+        self.ordered_quantity = ordered_quantity
+
+    def sell(self):
+        for bouquet in bouquets:
+            if bouquet['bouquet_name'] != self.bouquet_name:
+                continue
```

Comment:
Create a separate function(s) that will check whether or not we can sell / order something and if we can't raise a `ValueError` or another suitable exception